cedar-policy-validator/src/human_schema/grammar.lalrpop

```diff
@@ -199,7 +199,13 @@ Ident: Node<Id> = {
     <l:@L> STRING <r:@R> 
         => Node::with_source_loc("String".parse().unwrap(), Loc::new(l..r, Arc::clone(src))),
     <l:@L> <i:IDENTIFIER> <r:@R>
-        => Node::with_source_loc(i.parse().unwrap(), Loc::new(l..r, Arc::clone(src))),
```

Comment:
Calling this out as failure of our panic-audit process

Reply:
Yeah. Lalrpop uses panic internally in its generated code so we have to except it from the checking.